src/components/sidebar/PathSelector.tsx

```diff
@@ -225,52 +204,13 @@ class PathSelectorOption extends Component<OptionProps, OptionState> {
           </Tooltip>
         </div>
         {/* Settings part */}
-        {this.state.settingsOpen && (
+        {/* {this.state.settingsOpen && (
```

Comment:
Why keep this?

Reply:
Because this is a preparation PR ahead of adding a setting for target dt that will go in that panel.